content/docs/intro/vs/cloud_templates.md

```diff
@@ -11,21 +10,25 @@ menu:
 aliases: ["/docs/reference/vs/cloud_templates/"]
 ---
 
+## What is CloudFormation?
```

Comment:
This heading doesn't work, as the content addresses ARM also. IMO we should just remove it.

Reply:
👍